frontend/src/store/questions/index.js

```diff
@@ -37,7 +37,11 @@ const mutations = {
   FOCUS_ANSWERS (state, answers) {
     state.questionObj.answers = answers.map(answerMapper)
   },
-  API_ERROR (state, {code, message}) {
+  FOCUS_LIKES (state, { likes }) {
+    state.questionObj.question.likes = likes
```

Comment:
Can be handled in frontend logic I think.

Reply:
When you play around with this particular component to make it pretty feel free to transfer the logic, for now this is just a simple way to ensure you aren't doubling up on votes.